src/Symfony/Component/Security/Core/Resources/translations/security.zh_TW.xlf

```diff
@@ -16,67 +16,67 @@
             </trans-unit>
             <trans-unit id="4">
                 <source>Invalid credentials.</source>
-                <target>無效的憑證。</target>
+                <target>登入憑證無效。</target>
             </trans-unit>
             <trans-unit id="5">
                 <source>Cookie has already been used by someone else.</source>
                 <target>Cookie 已經被其他人使用。</target>
             </trans-unit>
             <trans-unit id="6">
                 <source>Not privileged to request the resource.</source>
-                <target>沒有權限請求此資源。</target>
+                <target>無權請求此資源。</target>
             </trans-unit>
             <trans-unit id="7">
                 <source>Invalid CSRF token.</source>
-                <target>無效的 CSRF token 。</target>
+                <target>無效的 CSRF token。</target>
             </trans-unit>
             <trans-unit id="9">
                 <source>No authentication provider found to support the authentication token.</source>
-                <target>沒有找到支持此 token 的身份驗證服務提供方。</target>
+                <target>沒有找到支援這個 token 的身分驗證服務提供方。</target>
```

Comment:
1. It seems unnecessary to change from "此" to "這個", but both are fine.
2. I think it should be "找不到" rather than "沒有找到" (or vice versa) to unify wording.

Reply:
```suggestion
                <target>找不到支援此 token 的身分驗證服務提供方。</target>
```